CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 2.0.5 (2022-12-22)
```

Comment:
Since you're creating a new patch version, this PR should be merged against `v2.0.5` instead of `v2.0.4` (also, if you can add the leading `v` into the changelogs...)

Reply:
Copy that.